src/components/VTable/VTable.vue

```diff
@@ -16,123 +16,126 @@
             )}
         </dl>
       </div>
-    </div> -->
+    </div>
+  -->
 
-    <div
-      ref="container"
-      class="table-container"
-      tabindex="0"
-      role="group"
-      aria-labelledby="caption"
-    >
-      <table>
-        <caption v-if="caption" id="caption">
-          {{ caption }}
-        </caption>
-        <thead v-if="headers.length">
-          <tr>
-            <th
-              v-for="(header, key) in cHeaders"
-              :key="key"
-              role="columnheader"
-              :aria-sort="
-                sortBy !== header.key
-                  ? null
-                  : sortOrder === 'ASC'
-                  ? 'ascending'
-                  : 'descending'
-              "
-            >
-              {{ header.text || header.key }}
+  <div
+    ref="container"
+    class="table-container"
+    tabindex="0"
+    role="group"
+    aria-labelledby="caption"
+  >
+    <table>
+      <caption
+        v-if="caption"
+        id="caption"
+      >
+        {{ caption }}
+      </caption>
+      <thead v-if="headers.length">
+        <tr>
+          <th
+            v-for="(header, key) in cHeaders"
+            :key="key"
+            :aria-sort="ariaSort(header)"
+          >
+            {{ header.text || header.key }}
 
-              <button
-                v-if="header.sortable"
-                :aria-label="
-                  `sort by ${header.text || header.key} in ${
-                    !sortOrder
-                      ? 'ascending'
-                      : sortOrder === 'ASC'
-                      ? 'descending'
-                      : 'default'
-                  } order`
-                "
-                @click="header.sortable && onSort(header.key)"
+            <button
+              v-if="header.sortable"
+              :aria-label="ariaLabel(header)"
+              @click="headerSort(header)"
+              @keyup.enter="headerSort(header)"
+              @keyup.space="headerSort(header)"
+            >
+              <template v-if="header.key === sortBy && sortOrder === 'ASC'">
+                &uarr;
+              </template>
+              <template
+                v-else-if="header.key === sortBy && sortOrder === 'DESC'"
               >
-                <template v-if="header.key === sortBy && sortOrder === 'ASC'">
-                  &uarr;
-                </template>
-                <template
-                  v-else-if="header.key === sortBy && sortOrder === 'DESC'"
+                &darr;
+              </template>
+              <template v-else>
+                ↕
+              </template>
+            </button>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <slot v-bind="{ items: cItems, ...$data, perPage }">
+          <tr
+            v-for="(item, index) in cItems"
+            :key="item.id"
+            @click="emitRowClick(item)"
+            @keyup="emitRowClick(item)"
```

Comment:
not certain about if a `<tr>` can receive focus to allow for a keyup event

Reply:
After trying it out, it works fine, you just need to add a `tabindex="0"` so it will be focus-able via keyboard. the `0` means it will follow the natural DOM order for tabbing.

Reply:
It cannot, by default. This feature was added by another contributor, but Im not sure how I feel about it. The VTable is something I need to use more to get a better feel for. The contributor suggested adding tabindex to table rows, but I think that is a big no for now. I dont want to force keyboard users to tab through potentially hundreds of table rows. 

That said, I also recognize that any click handler should be reachable by keyboard events. So this is one that needs more testing and thinking.